server/src/repositories/person.repository.ts

```diff
@@ -49,26 +49,23 @@ export class PersonRepository implements IPersonRepository {
     await this.personRepository.clear();
   }
 
-  async deleteAllFaces(sourceType?: string): Promise<void> {
-    await (sourceType
-      ? this.assetFaceRepository
-          .createQueryBuilder('asset_faces')
-          .delete()
-          .andWhere('sourceType = :sourceType', { sourceType: sourceType })
-          .execute()
-      : this.assetFaceRepository.query('TRUNCATE TABLE asset_faces CASCADE'));
-  }
-
-  async deleteFaceFromAsset(assetId: string, sourceType?: string): Promise<void> {
-    let query = this.assetFaceRepository
-      .createQueryBuilder('asset_faces')
-      .delete()
-      .where('assetId = :assetId', { assetId: assetId });
-
-    if (sourceType) {
-      query = query.andWhere('sourceType = :sourceType', { sourceType: sourceType });
+  async deleteAllFaces(sourceType: string | null): Promise<void> {
```

Comment:
Should still have the question mark since it can be undefined.

Reply:
Done - https://github.com/bugfest/immich/commit/bfd601c594308a09f3ccea25963230cd4bc3e8a0